packages/dashboard/src/components/pages/trade-type/cfd-test.tsx

```diff
@@ -0,0 +1,109 @@
+];
+
+const pair_data = [
+    {
+        title: 'Major pairs',
+        data: [
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'AUD/JPY' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'AUD/USD' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'EUR/AUD' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'EUR/CAD' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'EUR/CAD' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'EUR/CAD' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'EUR/CAD' },
+        ],
+    },
+
+    {
+        title: 'Minor pairs',
+        data: [
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'AUD/JPY' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'AUD/USD' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'EUR/AUD' },
+            { icon: 'IcUnderlyingFRXAUDJPY', description: 'EUR/CAD' },
+        ],
+    },
+];
+
+const indice_data = [
+    {
+        title: 'Volatility indices',
+        sub_title:
+            'These indices correspond to simulated markets with constant volatilities of 10%, 25%, 50%, 75%, and 100%. One tick is generated every two seconds for volatility indices 10, 25, 50, 75, and 100. One tick is generated every second for volatility indices 10 (1s), 25 (1s), 50 (1s), 75 (1s), and 100 (1s).',
+        line: true,
+        data: [
+            { icon: 'IcUnderlyingCRASH500', description: 'Volatility 10 Index' },
+            { icon: 'IcUnderlyingCRASH1000', description: 'Volatility 10 Index' },
+            { icon: 'IcUnderlyingBOOM1000', description: 'Volatility 50 Index' },
+            { icon: 'IcUnderlyingCRASH500', description: 'Volatility 75 Index' },
+            { icon: 'IcUnderlyingBOOM1000', description: 'Volatility 100 Index' },
+            { icon: 'IcUnderlyingCRASH500', description: 'Volatility 10 (1s) Index' },
+            { icon: 'IcUnderlyingCRASH500', description: 'Volatility 25 (1s) Index' },
+        ],
+    },
+
+    {
+        title: 'Crash/Boom',
+        sub_title:
+            'With these indices, there is an average of one drop (crash) or one spike (boom) in prices that occurs in a series of 1,000 or 500 ticks.',
+        data: [
+            { icon: 'IcUnderlyingCRASH500', description: 'Boom 1000 Index' },
+            { icon: 'IcUnderlyingCRASH500', description: 'Boom 500 Index' },
+            { icon: 'IcUnderlyingBOOM1000', description: 'Boom 1000 Index' },
+            { icon: 'IcUnderlyingCRASH500', description: 'Boom 5000 Index' },
+        ],
+    },
+];
+
+const CFDTest: React.FC = () => {
+    return (
+        <React.Fragment>
+            <div style={{ maxWidth: '93.6rem', margin: 'auto', backgroundColor: 'white' }}>
+                <Banner bg_image_url='trade-type/banner-background.png' type='Margin' title='Trade positions larger than your capital' />
```

Comment:
We also need to localize titles and descriptions on this page

Reply:
same as above :D